nb-api/src/main/java/io/nosqlbench/api/config/standard/NBConfiguration.java

```diff
@@ -41,13 +41,22 @@ protected NBConfiguration(NBConfigModel model, LinkedHashMap<String, Object> val
         this.model = model;
     }
 
+    public LinkedHashMap<String, Object> getData() {
```

Comment:
Let's hide this data and force type-safe access through model-checked accessors.